### DISCUSSION TITLE
roadmap for next release

### DISCUSSION THREAD

**Discussion by dscaravaggi** [rabbitmq/rabbitmq-dotnet-client#1046] [General]
Hi all,
I noticed that 6.2.1 is August 2020,
I tested lastest build and there are several optimizations, is there a schedulated date for next release ?

thanks a lot Diego

**Comment by sungam3r** [comment]
https://www.nuget.org/packages/RabbitMQ.Client/6.2.2

**Comment by lukebakken** [comment]
https://github.com/rabbitmq/rabbitmq-dotnet-client/milestones
